entries/a/ajax.systems.json

```
@@ -0,0 +1,12 @@
+{
+  "Ajax Systems": {
+    "domain": "ajax.systems",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.ajax.systems/en/how-secure-ajax-account/",
+    "categories": [
+      "other"
```

```suggestion
      "iot"
```

Other alarm systems have been tagged with the IoT category